flowmanager/datasets.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+import decimal
+import os
+
+import elasticsearch
+from concurrent.futures import ThreadPoolExecutor
+
+from tableschema_elasticsearch import Storage
+from datapackage_pipelines.utilities.extended_json import LazyJsonLine
+
+tpe = ThreadPoolExecutor(max_workers=1)
+
+ELASTICSEARCH_HOST = os.environ.get('EVENTS_ELASTICSEARCH_HOST', 'localhost:9200')
+DATASETS_INDEX_NAME = os.environ.get('DATASETS_INDEX_NAME', 'datahub')
+DATASETS_DOCTYPE = 'dataset'
+
+SCHEMA = {
+    'fields': [
+        {'name': 'id', 'type': 'string'},
```

Comment:
we should start using different analyzers for different fields:
https://www.elastic.co/guide/en/elasticsearch/reference/5.3/analysis-analyzers.html

e.g. for id, name etc. I would use keyword analyzer